`aiidalab registry build` fails when categories are specified in the setup.cfg

here is an example of a categories specification that doesn't work:

```
categories =
    quantum
```

The error is: 

```python
'\nquantum' is not of type 'array'

Failed validating 'type' in schema['properties']['apps']['patternProperties']['.*']['properties']['categories']:
    {'items': {'type': 'string'}, 'type': 'array'}

On instance['apps']['empa-molecules']['categories']:
    '\nquantum'
```

Also, this doesn't work:

```
categories = quantum
```

the error is:

```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/bin/aiidalab", line 8, in <module>
    sys.exit(cli())
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/__main__.py", line 678, in build
    build_registry(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/registry/web.py", line 97, in build
    for outfile in chain(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/registry/html.py", line 48, in build_html
    app_page_template.render(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "src/templates/app_page.html", line 1, in top-level template code
    {% extends "app_page_base.html" %}
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/registry/templates/app_page_base.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "src/templates/layout.html", line 4, in top-level template code
    {% extends "base.html" %}
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/registry/templates/base.html", line 35, in top-level template code
    {% block body %} {% endblock body %}
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiidalab/registry/templates/app_page_base.html", line 26, in block "body"
    <span class="category">{{ category_map[category]["title"] }}</span>
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jinja2/environment.py", line 452, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'q'
Error: Process completed with exit code 1.
```

--- a/aiidalab/metadata.py
+++ b/aiidalab/metadata.py
@@ -64,7 +64,12 @@ def _parse_setup_cfg(setup_cfg):
         "state", _map_development_state(metadata_pep426.get("classifiers", []))
     )
 
-    yield "categories", aiidalab.get("categories", [])
+    # Allow passing single category and convert to list
+    # and allow parse line separated string as list
+    categories = aiidalab.get("categories", [])
+    if isinstance(categories, str):
+        categories = [c for c in categories.split("\n") if c]
+    yield "categories", categories
 
 
 @dataclass

